pom.xml

```diff
@@ -217,6 +208,10 @@
           <preBuildHookScript>setup</preBuildHookScript>
           <postBuildHookScript>verify</postBuildHookScript>
           <addTestClassPath>true</addTestClassPath>
+          <properties>
+            <!-- e.g. ensure that Java7 picks up TLSv1.2 when connecting with Central -->
+            <https.protocols>TLSv1.2</https.protocols>
```

Comment:
Isn't this a duplicate setting with L23 above?

Reply:
Idea for that was from https://issues.apache.org/jira/browse/MPOM-188 - I hope soon it will not be required :)